kwok/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Start from a base image
+FROM registry.access.redhat.com/ubi8:latest
+
+# Update packages
+RUN yum update -y && yum clean all
```

Comment:
Never do this! it makes you image build unreproducible!

Reply:
removed